Potapov Vladislav/task1.js

```diff
@@ -0,0 +1,7 @@
+function splitAndMerge(str, sp) {
+    strWithoutSpaces = str.split(' ').join('');
+    return strWithoutSpaces.split('').join(sp);
+}
+
+console.log(splitAndMerge("My name is John"," "));
+console.log(splitAndMerge('Hello World!',','));
```

Comment:
@TheUnforgivenOne Please pay attention to description of the task and especially to the second example. We need to place separator only between letters in words, not between words